localstack.go

```diff
@@ -214,7 +223,7 @@ func (i *Instance) start(ctx context.Context) error {
 }
 
 func (i *Instance) startLocalstack(ctx context.Context) error {
-	imageName := "localstack/localstack:" + i.version
+	imageName := i.image + i.version
```

Comment:
there exist some tests which bypass the NewInstance which cause a nil pointer panic. Not sure on the best course of action here, could use an opinion as I'm still somewhat new to golang.

Reply:
Why don't you use a Pull Through Registry (https://docs.docker.com/registry/recipes/mirror/)? There's no guarantee that every docker image will work the same as localstack.